src/internal/components/chart-plot/index.tsx

```diff
@@ -114,12 +115,11 @@ function ChartPlot(
     plotClickedRef.current = true;
   };
   const onPlotFocus = (event: React.FocusEvent<SVGSVGElement>) => {
-    if (event.target === svgRef.current && !plotClickedRef.current) {
-      setPlotFocused(true);
-    }
-    // The click should focus the underling application bypassing the svg.
-    else if (plotClickedRef.current) {
+    // The click should focus the underlying application bypassing the svg.
+    if (plotClickedRef.current || !!activeElementKey) {
       applicationRef.current!.focus();
```

Comment:
Can we add a comment here explaining why we focus the application here?

And likewise a similar comment for the `else` case.

Reply:
Done